src/pages/InterviewSchedulingPage/components/SchedulersWithConfirmButton.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+
+import InterviewScheduler from './InterviewScheduler';
+import ConfirmScheduleButton from './ConfirmScheduleButton';
+
+interface SchedulersWithConfirmButtonProps {
+  startDates: Date[];
+}
+
+function SchedulersWithConfirmButton({
+  startDates,
+}: SchedulersWithConfirmButtonProps): JSX.Element {
+  const [dirty, setDirty] = useState(false);
+  const [userSchedules, setUserSchedules] = useState<Date[][]>([[]]);
+
+  useEffect(() => {
```

Comment:
can we push this useEffect to the page component for consistency? the other pages currently have useEffect + *most* state in the page level component afaik